internal/server/Spawner.go

```diff
@@ -22,7 +22,7 @@ func BikerAgentGenerator() objects.IBaseBiker {
 }
 
 func AudiAgentGenerator() objects.IBaseBiker {
```

Comment:
I don't know who wrote this originally (@harryjph ? ) but Audi should not be an agent. This should return IAudi rather than an IBaseBiker object

Reply:
Yep, I wrote this originally. This will be changed